src/newViews/UserGroup/index.tsx

```diff
@@ -0,0 +1,279 @@
+import React, { useMemo, useState, useCallback } from 'react';
+import { connect } from 'react-redux';
+import {
+    Button,
+    Container,
+    ContainerCard,
+    Pager,
+    PendingMessage,
+    Table,
+    TableColumn,
+    TableHeaderCellProps,
+    TableHeaderCell,
+    createStringColumn,
+    useRowExpansion,
+} from '@the-deep/deep-ui';
+import {
+    IoAdd,
+} from 'react-icons/io5';
+
+import { createDateColumn } from '#dui/tableHelpers';
+import {
+    activeUserSelector,
+} from '#redux';
+import {
+    useRequest,
+    useLazyRequest,
+} from '#utils/request';
+import {
+    AppState,
+    MultiResponse,
+} from '#typings';
+import ActionCell, { Props as ActionCellProps } from '#dui/EditDeleteActionCell';
+import { useModalState } from '#hooks/stateManagement';
+import _ts from '#ts';
+
+import AddUsergroupModal from './AddUsergroupModal';
+import styles from './styles.scss';
+
+const mapStateToProps = (state: AppState) => ({
+    activeUser: activeUserSelector(state),
+});
+
+interface Membership {
+    id: number;
+    member: number;
+    memberName: string;
+    memberEmail: string;
+    role: string;
+    group: number;
+    joinedAt: string;
+}
+
+interface Usergroup {
+    id: number;
+    title: string;
+    description: string;
+    role: string;
+    memberships: Membership[];
+    globalCrisisMonitoring: boolean;
+    createdAt: string;
+    modifiedAt: string;
+}
+
+interface UsergroupValueToSend {
+    title: string;
+    description?: string;
+    globalCrisisMonitoring?: boolean;
+}
+
+const MAX_ITEMS_PER_PAGE = 10;
+const usergroupKeySelector = (d:Usergroup) => d.id;
+
+interface Props {
+    activeUser: { userId: number };
+}
+
+function UserGroup(props: Props) {
+    const {
+        activeUser,
+    } = props;
+
+    const [activePage, setActivePage] = useState<number>(1);
+    const [
+        showAddUSerGroupModal,
+        setModalShow,
+        setModalHidden,
+    ] = useModalState(false);
+
+    const usergroupQuery = useMemo(() => ({
+        user: activeUser.userId,
+        offset: (activePage - 1) * MAX_ITEMS_PER_PAGE,
+        limit: MAX_ITEMS_PER_PAGE,
+    }), [activeUser.userId, activePage]);
+
+    const {
+        pending: usergroupGetPending,
+        response: usergroupResponse,
+    } = useRequest<MultiResponse<Usergroup>>({
+        url: 'server://user-groups/member-of/',
+        method: 'GET',
+        query: usergroupQuery,
+    });
+
+    const {
+        pending: postNewUsergroupPending,
+    } = useLazyRequest<unknown, UsergroupValueToSend>({
+        url: 'server://user-groups/',
+        method: 'POST',
+    });
+
+    console.warn(postNewUsergroupPending);
```

Comment:
Remove this. Also you've checked that there are no console.warns remaining. Please verify before sending.
> TIP: Review your PR before assigning.

Reply:
My bad!